src/Components/CommentBox/CommentBox.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+import DropDown from "Components/DropDown/DropDown";
+import { deleteComment, upVoteComment } from "Redux/Reducers-Redux/commentsSlice";
+import { useDispatch } from "react-redux";
+const CommentBox = ({ commentObj, postObj, setModalDisplay, setCommentId }) => {
+  const dispatch = useDispatch();
+  console.log(commentObj);
+  return (
+    <>
+      <div className="post">
+        <img
+          className="post__author-logo"
+          src={commentObj?.userPhoto}
+          al="user image"
```

Comment:
It should be alt attribute in img element

Reply:
my bad t was missed thanks for noticing